### PR TITLE
update cocoapods to 0.36.0.beta1 pre-release

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

**Comment by corinnekrych** [comment]
@cvasilak @passos 
this PR contains: 
- I've added a podfile to comply with other repos.
- travis build 
- cocoapods pre-release + readme

do you mind to review?

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,16 @@
 language: objective-c
-osx_image: xcode61
+osx_image: xcode611
 xcode_project: AeroGearJsonSZ.xcodeproj
 xcode_scheme: AeroGearJsonSZ
 xcode_sdk: iphonesimulator
 
+before_install:
+    - gem install cocoapods --pre --no-rdoc --no-ri --no-document --quiet
```
Comment (cvasilak) [inline .travis.yml]: if the Podfile is removed let's remove the before_install: step

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+
+script:
+    - set -o pipefail && xcodebuild -workspace AeroGearJsonSZ.xcworkspace -scheme AeroGearJsonSZ -sdk iphonesimulator -configuration Release clean test
```
Comment (cvasilak) [inline .travis.yml]: if xcodebuild is required to I am fine including it, if not let's remove it and use the provided xctool

File: README.md [hunk 3]
```diff
@@ -83,16 +83,16 @@ Last, we would like to give appreciation and credit to the existing serializatio
...
 
 and then:
+
 ```bash
-bundle install
-bundle exec pod install
+pod install
 ```
 
 to install your dependencies
```
Comment (cvasilak) [inline README.md]: let remove the 'Drag the library in your project'  and reuse [the instructions](https://github.com/corinnekrych/aerogear-ios-httpstub/blob/cocopods.0.36.support/README.md) we have already in the http-lib for the "Manual Installation"

**Comment by corinnekrych** [comment]
@cvasilak +1 I think having consistent README across repos make sense. 
With the same consistency effort, using workspace for all projects is better, same for using consistent travis script

File: Podfile [hunk 4]
```diff
@@ -0,0 +1,4 @@
+source 'https://github.com/CocoaPods/Specs.git'
+
+xcodeproj 'AeroGearJsonSZ.xcodeproj'
+platform :ios, '8.0'
```
Comment (danielpassos) [inline Podfile]: Is it a _empty_ podfile?

File: README.md [hunk 5]
```diff
@@ -66,55 +66,53 @@ println(user.first!)
...
 ```
 pod 'AeroGearJsonSZ'
 ```
 
 and then:
+
 ```bash
-bundle install
-bundle exec pod install
+pod install
```
Comment (danielpassos) [inline README.md]: Does not make sense for me try install a podfile without dependencies.

**Comment by cvasilak** [comment]
:+1: after the last commits

**Comment by cvasilak** [comment]
thank you @corinnekrych :rocket: landed on https://github.com/aerogear/aerogear-ios-jsonsz/commit/c93131191b385364ee42b22ce49d26ae903a5368

**Pull request merged**